### PR TITLE
zh: sync /docs/tasks/telemetry/distributed-tracing/overview/index.md

### PR DESCRIPTION
**Pull request by haiker2011**

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:pushed]`

Thank you @haiker2011, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/981) had been updated.

File: content_zh/docs/tasks/telemetry/distributed-tracing/overview/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ keywords: [telemetry,tracing]
 
 虽然 Istio 代理能够自动发送 span，但仍然需要一些线索来将整个追踪衔接起来。应用程序需要分发合适的 HTTP header，以便当代理发送 span 信息时，span 可以被正确的关联到一个追踪中。
 
-为此，应用程序需要收集传入请求中的 header，并将其传播到任何传出请求。header 如下所示：
+例如，如果您查看 Python 示例 `productpage` 服务，您将看到应用程序从 HTTP 请求中提取所需的头通过[OpenTracing](https://opentracing.io/)库：
```
Comment (rootsongjc) [inline content_zh/docs/tasks/telemetry/distributed-tracing/overview/index.md]: 英文两端需要加空格。

Comment (haiker2011) [reply content_zh/docs/tasks/telemetry/distributed-tracing/overview/index.md]: 已经更新

**Comment by fleeto** [comment]
`[trans-bot:merged]`

Thank you @haiker2011, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/981) had been updated.

**Pull request merged**